### PR TITLE
mengupdate readme

### PR DESCRIPTION
**Pull request by Azissa**
Bang Rizal,tolong di review dan kasih sarannya kalo ada yang kurang readme-nya

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+## Brief Description
+This repository contains the source code and documentation for the creation of the To-Do List App. This app is designed to help users manage and organize their daily tasks in an efficient and effective way.
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+### how to contribute
+1. Open terminal and clone this repository to your machine local using `git clone repositorylink`.
+2. go to the project directory wtih `cd repo-name`.
+3. Install the dependencies by using Yarn `yarn install
+`.
+4. Run the application with the Yarn command `yarn start`.
```
Comment (pahrizal) [inline README.md]: @Azissa do we have this `start` command in `scripts` section of `package.json` file?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
-Currently, two official plugins are available:
-
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+### Contribution
+- Pahrizal Marup
+- Dharma Chandra Viriya
+- Juhanda Azril Kamal
+- Chandra Kusuma
+- Azis Satya Agni
```
Comment (pahrizal) [inline README.md]: I think we don't need this section, since github will automatically track this for us

File: README.md (hunk 1 above)
Comment (Azissa) [reply README.md]: no sir,do i have to delete the no.4 and replace it with something like `git branch` and `git push`?

File: README.md (hunk 2 above)
Comment (Azissa) [reply README.md]: okay sir

File: README.md (hunk 1 above)
Comment (pahrizal) [reply README.md]: @Azissa no. I mean, use one of command listed in that section. the command should be `yarn dev`

Comment (Azissa) [reply README.md]: okay got it sir

**Review by pahrizal — APPROVED** [review]
Looking good! 🎉

**Pull request merged**